applications/nrf_desktop/doc/ble_state.rst

```diff
@@ -23,14 +23,19 @@ Module events
 .. note::
     |nrf_desktop_module_event_note|
 
-Implementation details
-**********************
+Configuration
+*************
 
 nRF Desktop uses the Bluetooth LE state module from :ref:`lib_caf` (CAF).
-See the :ref:`CAF Bluetooth LE state <caf_ble_state>` page for implementation details.
+The :ref:`CONFIG_DESKTOP_BLE_STATE <config_desktop_app_options>` Kconfig option selects :kconfig:option:`CONFIG_CAF_BLE_STATE`.
+
+For more information about Bluetooth configuration in nRF Desktop, see :ref:`nrf_desktop_bluetooth_guide`.
```

Comment:
This kind of documentation improvement (moving from section to section), not directly related to the functional change, should be separated into a different commit (e.g. doc clean-up).

Alternatively, you can separate the functional changes (Kconfig changes) from the documentation changes.

However, we may want to rethink the approach of having the follow-up documentation changes in the separate PRs and commits as it makes it hard to revert the changes (functional and documentation changes are not contained in the single commit).

Reply:
(This comment also applies to other parts of the documentation).

Reply:
The change introduced new Kconfig options `CONFIG_DESKTOP_caf_module_name` and the related doc update is added together with the code. The small documentation structure alignment actually reflects changes in the code (that's why the changes were added together as a single commit).
If you prefer I can split the changes, but then (as you mentioned) doc change will be separated from code.

Reply:
I could also simply update a commit message to explicitly inform that it introduces these new Kconfig options.